priority: use SF terms, not ABNF (#1977)
httpwg/http-extensions: draft-ietf-httpbis-priority.md

{::boilerplate bcp14-tagged}

> The terms Boolean, Dictionary, and Integer, along with the sf-dictionary rule,

Reviewer: I don't think that you want to say that RFC 8941 defines the term "Boolean".  Maybe instead "This document uses the Boolean, Dictionary, and Integer types from {{!SF}} along with the `sf-dictionary` ABNF rule."

Author: Thanks. "Boolean" is so generic, it was part of the reason why I gravitated to `sf-boolean` in prose in the first place.

Reviewer 2: In prose, you can try "Boolean Structured Type" or similar.

Author: The pattern I've seen so far is people saying "Structured Fields Dictionary" or "Structured Fields List". So "Structured Fields Boolean" would be consistent with that but its a bit wordy. Perhaps settling on a common acronym of `SF` in front of these would work.

Author: Applying this to "SF Member" would also be help in specs that need to talk generically about the things in List or Dictionary without wanting to have to make up a term of their own.

## Incremental

> The incremental parameter (`i`) takes a Boolean (see
> {{Section 3.3.6 of STRUCTURED-FIELDS}}) as the value that indicates

Reviewer 3: ```suggestion
The incremental parameter (`i`) is a Boolean (see
{{Section 3.3.6 of STRUCTURED-FIELDS}}) that indicates
```
